components/Footer/Footer.tsx

```diff
@@ -55,70 +54,21 @@ const Footer = ({
 		scrollToRef(ref)
 	}
 
+
 	return (
-		<div className="footer">
+		<div className="bg-zircon-50 grid grid-cols-3 lg:grid-cols-5 gap-4 px-6 lg:px-[150px] py-6 items-center">
```

Comment:
This line is 113 characters long, the linting limit is 110, please sort this? I know it sounds picky sorry.